src/main/java/seedu/address/model/assignment/DueDate.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.assignment;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class DueDate {
+    public static final String MESSAGE_CONSTRAINTS = "Due dates should be in a format dd/MM/yyyy HHmm";
+    //TODO: decide which parse, output format to keep
+    protected static final DateTimeFormatter PARSE_FORMAT = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
+    protected static final DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("dd MMM yyyy, hh:mm a");
+    //TODO: change VALIDATION INDEX
+    //public static final String VALIDATION_REGEX = "\\d{3,}";
+    public final LocalDateTime value;
+
+    /**
+     * Constructs a {@code DueDate}.
+     *
+     * @param dateTime A valid dateTime.
+     */
+    public DueDate(LocalDateTime dateTime) {
+        requireNonNull(dateTime);
+        // TODO: use Validation regex to verify dateTime format
+        //checkArgument(isValidPhone(phone), MESSAGE_CONSTRAINTS);
```

Comment:
maybe u should include this to validate the due date form

Reply:
This is related to the ```VALIDATION_REGEX``` in the previous review. If you are ok with the method body I suggest, I can uncomment this ```checkArgument()``` method.